src/views/AmalgamationSelection.vue

```diff
@@ -220,17 +222,25 @@ export default class AmalgamationSelection extends Vue {
    * @returns the business identifier of the newly created amalgamation application
    */
   private async createBusinessAA (type: AmalgamationTypes): Promise<string> {
-    const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
-    const email = this.isShortFormAmalgamation(type) ? this.getBusinessEmail : ''
-    const phone = this.isShortFormAmalgamation(type) ? this.getFullPhoneNumber : ''
-    const legalName = this.isShortFormAmalgamation(type) ? this.getLegalName : ''
-    const correctNameOption = this.isShortFormAmalgamation(type) ? CorrectNameOptions.CORRECT_AML_ADOPT : null
+    // short-form amalgamations use the legal name of the primary business,
+    // otherwise don't set a legal name
+    const legalName = this.isShortFormAmalgamation(type) ? this.getLegalName : undefined
+
+    // short-form amalgamations adopt the name of the primary business,
+    // otherwise don't set a correct name option
+    const correctNameOption =
+      this.isShortFormAmalgamation(type) ? CorrectNameOptions.CORRECT_AML_ADOPT : undefined
+
+    // short-form amalgamations use the email and phone of the primary business (if they exist),
+    // otherwise set empty initial values
+    const email = (this.isShortFormAmalgamation(type) && this.getBusinessEmail) || ''
+    const phone = (this.isShortFormAmalgamation(type) && this.getFullPhoneNumber) || ''
```

Comment:
The old code assigned `''` here for non-short form amalgamation, and fell back to `''` below if getBusinessEmail or getFullPhoneNumber returned falsy.

I just combined it all here. Example:
| isShortFormAmalgamation | getBusinessEmail | result |
| - | - | - |
| true | '123' | '123' |
| true | '' | '' |
| false | '123' | '' |
| false | '' | '' |